.github/CONTRIBUTING.md

```diff
@@ -8,8 +8,9 @@ You will need to set some environment variables in a custom `phpunit.xml` file i
 
 Copy the default file using `cp phpunit.xml.dist phpunit.xml` and add the following lines below the `DB_CONNECTION` environment variable in your new `phpunit.xml` file:
 
-    <server name="PADDLE_VENDOR_ID" value="Your Paddle vendor ID"/>
-    <server name="PADDLE_VENDOR_AUTH_CODE" value="Your Paddle auth code"/>
-    <server name="PADDLE_TEST_PRODUCT" value="Identifier for a random one off product"/>
+    <env name="PADDLE_SANDBOX" value="true"/>
```

Comment:
I noted this in the PR but just calling it out here...I found that adding `<server>` did nothing but `<env>` got the result of injecting the environment variable for the tests. I also added SANDBOX to this too because it's likely that you'll be running tests in the sandbox...lemme know if you want me to yank it.

Reply:
This lgtm 👍